modules/customizing-the-web-console-URL.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * web_console/customizing-the-web-console.adoc
+
+[id="customizing-the-web-console-url_{context}"]
+= Customizing the web console URL
+
+To update the web console URL, `consoleURL`, to a custom value:
+
+. Modify the cluster instance created by default during installation in the `consoles.operator.openshift.io` custom resource:
++
+[source,terminal]
+----
+$ oc patch consoles.operator.openshift.io cluster --patch '{"spec":{"route":{"hostname":"console.example.com"}}}' --type=merge
+----
+
+. (Optional) If you specify a custom certificate, you must create a secret in the `openshift-config` namespace that has the key and certificate. For example:
```

Comment:
```suggestion
. If you specify a custom certificate, you must create a secret in the `openshift-config` namespace that has the key and certificate. For example:
```
It looks like this step is required in a certain context, which you've explained well, so it's not really optional.